pipelines/rbsa/tasks/normalize_loadshapes.py

```diff
@@ -0,0 +1,90 @@
+import logging
+import pandas as pd
+from generics import task as t
+from generics.file_type_enum import SupportedFileReadType
+
+
+logger = logging.getLogger('LCTK_APPLICATION_LOGGER')
+
+
+class NormalizeLoadshapes(t.Task):
+    """ 
+    This class is used to group sites into 3 digit zip codes
+    """
+    def __init__(self, name):
+        super().__init__(self)
+        self.name = name
+        self.input_artifact_enduse_loadshapes = 'enduse_loadshapes.csv'
+        self.output_artifact_normal_loadshapes = 'normal_loadshapes.csv'
+        self.my_data_files = [{ 'name': self.input_artifact_enduse_loadshapes, 'read_type': SupportedFileReadType.DATA }]
+        self.task_function = self._task
+
+    def _get_data(self):
+        return self.load_data(self.my_data_files)
+        
+    def _task(self):
+        data_map = self._get_data()
+        
+        self.df = data_map[self.input_artifact_enduse_loadshapes]
+
+        if self.df.columns[0] == 'Unnamed: 0':
+             self.df = self.df.drop('Unnamed: 0', axis=1)
+
+         # output dataframe initialization
+        initialization = True
+
+        city_list = self.df.target.unique()
+
+        for city in city_list:
+
+            city_df = self.df.loc[self.df.target == city]
+            city_df = city_df.set_index('time')
+
+            self.enduse_cols = list(city_df.columns)
+            self.enduse_cols.remove('target')
+            self.enduse_cols.remove('daytype')
+
+            # get enduse columns
+            normalization_val = self.get_normalization_val(city_df, summer=True)
+
+            # Normalize by peak total
+            city_df[self.enduse_cols] = city_df[self.enduse_cols]/normalization_val
+
+            # output dataframe 
+            if initialization:
+                normal_loadshapes = city_df
+                initialization = False
+            else:
+                normal_loadshapes = normal_loadshapes.append(city_df)
+
+        self.validate(normal_loadshapes)
+        self.on_complete({self.output_artifact_normal_loadshapes: normal_loadshapes})
+
+    def get_normalization_val(self, city_df, summer):
+        """
+        returns peak total, if summer is True returns peak summer 
+        """        
+        if summer:
+            city_df = city_df.loc[city_df['daytype'] == 'summer_peak']
+
+        totals = city_df[self.enduse_cols].sum(axis=1)
+        normalization_val = totals.max() # can be adjusted if normalizing for summer peak
+
+        # can check here for totals.idxmax() month to confirm summer/winter
+
+        return normalization_val
```

Comment:
Remove line 71 and just return `totals.max()`

Also, put the comment in line 73 in the docstring

Reply:
fixed